packages/sdk/package.json

```diff
@@ -42,8 +42,11 @@
     "@uma/contracts-frontend": "^0.1.1",
     "@uma/contracts-node": "^0.1.1",
     "axios": "^0.21.4",
+    "bn.js": "^5.2.0",
+    "decimal.js": "^10.3.1",
     "highland": "^2.13.5",
-    "lodash-es": "^4.17.21"
+    "lodash-es": "^4.17.21",
+    "web3": "^1.5.3"
```

Comment:
I think we should refactor this to not use web3.js! I originally did but ran into some snags. you'll basically need to implement your own. Just spitballing here but it will likely look something like this:

```
const toBN = (number: string | number) => new BN(number.toString());
const toWei = (number: string | number) => new BN(number.toString()).mul(new BN("1e18")).toString();
const toBNWei = (number: string | number) => toBN(toWei(number.toString()).toString());
```

Reply:
will look into this

Reply:
im going to do this in a different PR. having issues getting tests to pass in this form.
![image](https://user-images.githubusercontent.com/4429761/135500816-fb2ec9c6-9d27-4d88-968e-feecc2a6ef08.png)

Reply:
Agreed, would be nice to trim the dep tree.

Reply:
i had copied and pasted from gh, which caused the issue as matt pointed out. still having problems getting tests to pass for other reasons, so i will still implmeent in separate pr.